src/core/format-utils.js

```diff
@@ -0,0 +1,43 @@
+import BigNumber from 'bignumber.js';
```

Comment:
feels like this file would benefit from docs describing with examples what the functions do